README.md

```
@@ -193,6 +193,75 @@ use BEAR\ToolUse\Attribute\Tool;
+
+### Bind in DI Module
+
+```php
+$this->bind(ConfirmationHandlerInterface::class)->to(CliConfirmationHandler::class);
+```
+
+### How It Works
+
+The LLM's text response serves as the confirmation message. No templates needed.
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)